src/components/AgreementLineSections/FormCoverage/FormCoverage.test.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import '@folio/stripes-erm-components/test/jest/__mock__';
+import { TestForm, renderWithIntl } from '@folio/stripes-erm-components/test/jest/helpers';
+import { Accordion, KeyValue } from '@folio/stripes-testing';
+import FormCoverage from './FormCoverage';
+import { line, values, resource, handlers, initialValues } from './testResources';
+import translationsProperties from '../../../../test/helpers';
+
+jest.mock('../../CoverageFieldArray', () => () => <div>CoverageFieldArray</div>);
+jest.mock('@folio/stripes-erm-components', () => ({
+  ...jest.requireActual('@folio/stripes-erm-components'),
+  SerialCoverage: () => <div>SerialCoverage</div>,
+}));
+
+const onSubmit = jest.fn();
+
+describe('FormCoverage', () => {
+  let renderComponent;
+  describe('with no initialValues', () => {
+    beforeEach(() => {
+      renderComponent = renderWithIntl(
+        <TestForm onSubmit={onSubmit}>
+          <FormCoverage
+            line={line}
+            resource={resource}
+            values={values}
+          />
+        </TestForm>,
+        translationsProperties
+      );
+    });
+
+    test('renders the Coverage accordion', async () => {
+      await Accordion('Coverage').exists();
+    });
+
+    test('renders the expected Default coverage value', async () => {
+      await KeyValue('Default coverage').has({ value: 'SerialCoverage' });
+    });
+
+    test('renders the SerialCoverage component', () => {
+      const { getByText } = renderComponent;
+      expect(getByText('SerialCoverage')).toBeInTheDocument();
+    });
+
+    test('renders the expected embargo value', async () => {
+      await KeyValue('Embargo').has({ value: 'Moving wall end: 4 years' });
+    });
+  });
+
+  describe('with initialValues', () => {
```

Comment:
can we add more tests to this block, i.e. what is being rendered alongside the accordion and the CoverageFieldArray component when initialValues are passed, Embargo, SerialCoverage? etc?